apps/api/src/persistence/database/arango-database.ts

```diff
@@ -315,7 +315,7 @@ export class ArangoDatabase {
 
     // TODO We only want this power within test utilities!
     deleteAll = async (collectionName: string): Promise<void> => {
-        if (process.env.NODE_ENV !== Environment.test) {
+        if (process.env.NODE_ENV !== Environment.test && process.env['DATA_MODE'] !== '_CYPRESS_') {
```

Comment:
Nice to have a couple layers of security

Reply:
Yea. There is never a reason to empty out a production db. You can simply make a new db and update the .env to point to that one and the initialize script will add all the empty collections. But it's super useful for tests to not have to drop the db and add a new one with the same name which is not always possible depending on the level your working at.